### PR TITLE
you can slash flashlights off

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Make flashlights slashable. 

## Why It's Good For The Game

image
image

No thanks. Also makes sense since you can slash flashlights on gun

## Changelog
:cl:
balance: you can slash flashlights off
/:cl:

### CODE REVIEW

**Review by TiviPlus — APPROVED** [review]

File: code/game/objects/items/flashlight.dm [hunk 1]
```diff
@@ -37,6 +37,12 @@
 	update_action_button_icons()
 	update_icon()
 
+/obj/item/flashlight/attack_alien(mob/living/carbon/xenomorph/X, isrightclick = FALSE)
+	turn_light(X, FALSE)
```
Comment (QualityVan) [inline code/game/objects/items/flashlight.dm:41]:
```suggestion
/obj/item/flashlight/attack_alien(mob/living/carbon/xenomorph/X, isrightclick = FALSE)
	if(!light_on)
		return ..()
	turn_light(X, FALSE)
```

Comment (BraveMole) [reply code/game/objects/items/flashlight.dm:41]: image

Comment (QualityVan) [reply code/game/objects/items/flashlight.dm:41]: yeah but you'll get the attack animation and to_chat regardless

**Comment by EvilMelonTheSecond** [comment]
Aren't they just flares that are harder to carry? I don't see a problem with flashlight spam, melt em like flares.

**Comment by bowlofcereal** [comment]
I'm fine with this but I just gotta say that the hivelord in that image is a total dunce if he's not gonna even try to melt them

**Comment by Losenis** [comment]
> Aren't they just flares that are harder to carry? I don't see a problem with flashlight spam, melt em like flares.

They have the same slot size and are permanent so long as they don't get acided, as well as being infinite throught the vendor. A backpack of them will essentially last you the op since you can also just go and pick up whatever flashlight is lying around not being useful.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**